File: codetune/samples/sample.py
def square(x):
    return x * x


def clamp(a, b):
    x = a + b
    y = square(x)
    if y > 100:
        z = 100
    else:
        z = y
    w = 0
    while w < 3:
        w = w + 1
    return z

File: codetune/samples/sample.java
class Sample {
    int square(int x) {
        return x * x;
    }

    int clamp(int a, int b) {
        int x = a + b;
        int y = square(x);
        int z;
        if (y > 100) {
            z = 100;
        } else {
            z = y;
        }
        int w = 0;
        while (w < 3) {
            w = w + 1;
        }
        return z;
    }
}
